src/Symfony/Component/DependencyInjection/Dumper/YamlDumper.php

```diff
@@ -266,6 +266,7 @@ private function getServiceCall(string $id, Reference $reference = null): string
     {
         if (null !== $reference) {
             switch ($reference->getInvalidBehavior()) {
+                case ContainerInterface::RUNTIME_EXCEPTION_ON_INVALID_DEFINITION:
```

Comment:
I would break immediately instead of doing a fallthrough to the break in the next case.

Reply:
updated